lib/modules/apostrophe-schemas/public/css/components/array-editor.less

```diff
@@ -28,16 +28,26 @@
 // TODO re-figure .apos-modal .apos-schema-group {} width when no tabs are present.
 // Question: Do we need tabs in arrays?
 
-
 // "Add Item" button
 .apos-modal
-.apos-array-add-item
 {
-  position: absolute;
-  bottom: 0;
-  left: 0;
-  width: 15%;
-  @media @apos-breakpoint-desktop-xl { width: 25%; }
-  text-align: center;
-  padding: @apos-padding-2*.75 0;
+  .apos-array-add-item, .apos-array-limit-reached
+  {
+    position: absolute;
+    bottom: 0;
+    left: 0;
+    width: 15%;
+    @media @apos-breakpoint-desktop-xl { width: 25%; }
+    text-align: center;
+    padding: @apos-padding-2*.75 0;
+  }
+  .apos-array-limit-reached
+  {
+    display: none;
+    background-color: @apos-red;
```

Comment:
Do we want this to feel like an error? I think a red error state might be too strong in this case. Thoughts?